T14/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿internal class Program {
+   static void Main (string[] args) {
+      Console.WriteLine ("Enter the word");
+      string input = Console.ReadLine ().ToLower ();
+      if (!input.All (char.IsLetter)) {
+         Console.WriteLine ("Invaild input");
+         return;
+      }
+      int count, inputLen = input.Length;
+      List<char> inputCharList = new ();
+      for (int i = 97; i <= 123; i++) {

```

Comment:
123 ascii corresponds to { and it is not letter.
```suggestion
      for (int i = 97; i < 123; i++) {

```